src/main/java/mate/academy/Main.java

```diff
@@ -6,15 +6,21 @@
 import mate.academy.service.AuthenticationServiceImpl;
 import mate.academy.service.OrderService;
 import mate.academy.service.OrderServiceImpl;
+import org.apache.logging.log4j.LogManager;
+import org.apache.logging.log4j.Logger;
 
 public class Main {
+    private static final Logger logger = LogManager.getLogger(Main.class);
+
     public static void main(String[] args) {
+
         AuthenticationService authenticationService = new AuthenticationServiceImpl();
         User user;
         try {
             user = authenticationService.login("bob", "1234");
         } catch (AuthenticationException e) {
-            e.printStackTrace();
+            logger.error("Authentication failed for user: {}. Error message: {}",
```

Comment:
According to the checklist, log messages should be more informative. It's a good practice to include parameters in the log messages. In this case, you can include the user name in the log message. Here is a better example: `logger.error("Authentication failed for user: {}. Error message: {}", "bob", e.getMessage(), e);`